collect_data.py

```diff
@@ -2,22 +2,28 @@
 
 from sensors import *
 from writers import *
-
+from handlers import *
+from server import Server
 
 def main():
+    vid_handler = VideoHandler()
+    key_handler = KeyHandler()
     with Webcam() as cam, \
-         ControllerServer(port=8000) as controller, \
-         VideoServer(port=8050) as video_server, \
-         ImageDiskWriter(folder="collected_data") as img_disk_writer, \
-         CSVDiskWriter(filename="collected_data/classes.csv") as csv_disk_writer:
+        Server(handlers={
+            "/": FileHandler("index.html"),
```

Comment:
I would prefer to have `FileHandler(…)` above with the other two handlers to keep it consistent.